docs(README): Update README (#7211)
camunda/camunda-docs: howtos/setup.md

npm run start
```

### Clear Docusaurus cache

> You can clear Docusaurus cache running the following command:

Reviewer: ```suggestion
To clear the Docusaurus cache, run the following command:
```

```
npm run docusaurus -- clear
```

:::tip

> When renaming a file or changing its `id`, the continuous development server started with `npm run start` may fail due to cached content. Clearing the cache and then restarting the server will resolve this issue.

Reviewer: ```suggestion
When renaming a file or changing its `id`, the continuous development server started with `npm run start` can fail due to cached content. To resolve this issue, clear the cache and then restart the server.
```
